Controller/NotifyHandler.php

```diff
@@ -292,4 +295,53 @@ private function _shouldUseTransactionID($sTransactionId, $sPaymentId)
     {
         return is_null($sTransactionId) || $sPaymentId === CreditCardPaymentMethod::getName(true);
     }
+
+    /**
+     * Handle credit card
+     *
+     * @param SuccessResponse $oResponse
+     * @param BackendService  $oBackendService
+     * @param Order           $oOrder
+     *
+     * @return void
+     *
+     * @since 1.3.0
+     */
+    private static function _handleCreditCard($oResponse, $oBackendService, $oOrder)
+    {
+        $oLogger = Registry::getLogger();
+
+        if (!self::_shouldSaveCard($oResponse, $oOrder)) {
+            return;
+        }
+
+        try {
+            $aTransaction = $oBackendService->getTransactionByTransactionId(
+                $oResponse->getTransactionId(),
+                CreditCardTransaction::NAME
+            );
+
+            $oVault = new Vault();
+            $oVault->saveCard($oResponse, $aTransaction['payment']['card'], $oOrder);
+        } catch (DatabaseConnectionException $oExc) {
+            $oLogger->error("Cannot save card", [$oExc]);
+        } catch (DatabaseErrorException $oExc) {
+            $oLogger->error("Cannot save card", [$oExc]);
+        }
+    }
+
+    /**
+     * @param SuccessResponse $oResponse
+     * @param Order           $oOrder
+     *
+     * @return bool
+     *
+     * @since 1.3.0
+     */
+    private static function _shouldSaveCard($oResponse, $oOrder)
```

Comment:
Please add a method description on what you are checking for. Like Check if the payment method is credit card the user has an account and ...

Reply:
I added a description